api/lib/infrastructure/utils/csv/certification-results.js

```diff
@@ -72,7 +79,14 @@ function _buildCertificationResultsFileDataWithCertificationCenterName({ session
   return certificationResults.map(_getRowItemsFromSessionAndResults(session));
 }
 
-function _buildCertificationResultsFileHeadersWithCertificationCenterName() {
+function _buildCertificationResultsFileHeadersWithCertificationCenterName({ certificationResults }) {
+
+  const doesAtLeastOneCertificationHasPassedClea = certificationResults.some((result) => _hasPassedClea(result.cleaCertificationStatus));
```

Comment:
C'est bien d'avoir nommé correctement la variable, ça aurait été bien d'aller jusqu'à encapsuler le `some` dans une fonction privée `_doesAtLeastOneCertificationHasPassedClea(certificationResults)`.

Je pense que "_passed_" c'est trompeur pour ce que vous essayer de chercker en anglais ça veut dire "avoir obtenu" alors que vous essayez de savoir s'il l'ont "passé" qui pourrait être `_didAtLeastOneCandidateTakeClea` (`to take an exam`)